samples/SkiaAwtSample/src/main/kotlin/SkiaAwtSample/ResizableSwingSkia.kt

```diff
@@ -0,0 +1,50 @@
+package SkiaAwtSample
+
+import org.jetbrains.skia.Canvas
+import org.jetbrains.skia.Paint
+import org.jetbrains.skia.Rect
+import org.jetbrains.skiko.GenericSkikoView
+import org.jetbrains.skiko.SkikoView
+import java.awt.BorderLayout
+import java.awt.Color
+import javax.swing.*
+
+fun swingSkiaResize(
+    isStatic: Boolean,
+    offScreenRendering: Boolean
+) = SwingUtilities.invokeLater {
```

Comment:
I don't think you really need the invokeLater here. Button click listeners are called on the AWT thread.

Reply:
The same for `SkiaAwtSample.AppKt#createWindow`